std/haxe/io/Bytes.hx

```diff
@@ -348,6 +348,60 @@ class Bytes {
 
 	public function getString( pos : Int, len : Int, ?encoding : Encoding ) : String {
 		if( encoding == null ) encoding == UTF8;
+		if( encoding == UTF8Strict ) {
+			var s = "";
+			var b = b;
+			var fcc = String.fromCharCode;
```

Comment:
it's better to 
```haxe
inline function fcc(c) return String.fromCharCode(c);
```
This way it will be faster on some targets.

Reply:
This was in the original implementation, but sure, I'll change it.